### PR TITLE
Bouncing arrow scroll

### PR DESCRIPTION
**Pull request by jackcarlisle**
## ready for review

* adds smooth scroll to the bouncing arrow

### CODE REVIEW

File: lib/hero-image-header/index.js [hunk 1]
```diff
@@ -1,10 +1,20 @@
 import React, { Component, PropTypes } from 'react';
+var Scroll = require('react-scroll');
```
Comment (nelsonic) [inline lib/hero-image-header/index.js]: I would *suggest* adding some sort of detail in an issue or readme as to why we chose to use https://github.com/fisshy/react-scroll (*despite it's lack of tests...*), are we relying on the fact that it has a few thousand users to file issues and debug it if there is a browser incompatibility...? It works *now*...

File: lib/hero-image-header/index.js [hunk 2]
```diff
@@ -1,10 +1,20 @@
...
 import SearchBar from '../search-bar';
 export default class Header extends Component {
+  constructor () {
+    super();
+    this.state = {
+      gridItemsDisplayed: true
+    };
+  }
+  handleOnArrowClick () {
+    scroll.scrollTo(window.innerHeight / 1.05);
```
Comment (nelsonic) [inline lib/hero-image-header/index.js]: this *should* work reliably. 👍

**Comment by nelsonic** [comment]
Code looks good. 
Biggest change is in the schema. (`test/constants/helpers/schema.json`)
*Do* we want to add a test for the functionality before merge...?

**Comment by jackcarlisle** [comment]
Yeah I'll add some tests now!

File: lib/hero-image-header/index.js (hunk 1 above)
Comment (nikhilaravi) [inline lib/hero-image-header/index.js]: I guess we should have checked that - basically we needed a way of scrolling smoothly. We can look for a different module if necessary.

File: lib/hero-image-header/test/index.test.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ import Header from '../';
 describe('Component', function () {
   describe('<Header />', function () {
     it('Should render Header component', function (done) {
-      const wrapper = shallow(<Header />);
+      const wrapper = shallow(<Header displayedItems={[]}/>);
       expect(wrapper.find('.headerBarWrapper')).to.have.length(1);
```
Comment (nikhilaravi) [inline lib/hero-image-header/test/index.test.js]: You could also easily add a test that checks the arrow is displayed when the displayedItems array has items!

File: lib/hero-image-header/index.js (hunk 1 above)
Comment (nelsonic) [inline lib/hero-image-header/index.js]: Yes, we all _understand_ the smooth scrolling _requirement_, but right now it's not traceable.
it's *always* a good idea to leave a decision trail in the form of an issue which can be referred to in the readme... simply capturing the thought process that went into selecting *this* module should not be more than a couple of lines (*unless you want to go into detail...*)

I have a *suggestion*: 
let's merge this in now, and `then` as part of debugging https://github.com/numo-labs/isearch-ui/issues/189 (*scrolling bug*) we write a few lines of docs.

**Comment by nikhilaravi** [comment]
great!

**Pull request merged**